apisix/plugins/ai-prompt-decorator.lua

```diff
@@ -114,4 +114,5 @@ function _M.rewrite(conf, ctx)
 end
 
 
+_M.__decorate = decorate -- for ai-rag plugin
```

Comment:
it is unsafe, each plugin should be independent.

if different plugins are using same function, we should push this function to public library

Reply:
I found a better way to augument the request body.